spec/worker_spec.rb

```diff
@@ -120,56 +145,24 @@
       EOJ
     end
 
-    let(:sources_1_application_types_response) do
-      <<~EOJ
-        {
-          "links": {},
-          "data": [
-            {"id":"1","name":"/insights/platform/catalog","display_name":"Catalog"}
-          ]
-        }
-      EOJ
-    end
-
-    let(:sources_2_application_types_response) do
-      <<~EOJ
-        {
-          "links": {},
-          "data": [
-            {"id":"1","name":"/insights/platform/catalog","display_name":"Catalog"}
-          ]
-        }
-      EOJ
-    end
-
-    let(:sources_3_application_types_response) do
-      <<~EOJ
-        {
-          "links": {},
-          "data": [
-            {"id":"1","name":"/insights/platform/cost-management","display_name":"Cost Management"}
-          ]
-        }
-      EOJ
-    end
-
     it "generates the expected hash" do
       stub_rest_get("#{sources_api}/source_types", orchestrator_tenant_header, source_types_response)
+      stub_rest_get("#{sources_api}/application_types", orchestrator_tenant_header, application_types_response)
       stub_rest_get("http://topology.local:8080/internal/v1.0/tenants", orchestrator_tenant_header, tenants_response)
+
+      application_type_query = "filter%5Bapplication_type_id%5D%5Beq%5D%3D1%26filter%5Bapplication_type_id%5D%5Beq%5D%3D3"
```

Comment:
Should be `"filter%5Bapplication_type_id%5D%5Beq%5D%5B%5D%3D1%26filter%5Bapplication_type_id%5D%5Beq%5D%5B%5D%3D3"`